src/lib/components/Swap/RoutingDiagram/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Protocol } from '@uniswap/router-sdk'
+import { Currency, Percent, TradeType } from '@uniswap/sdk-core'
+import { Pair } from '@uniswap/v2-sdk'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { css } from 'lib/theme'
+import { InterfaceTrade } from 'state/routing/types'
+
+export interface RoutingDiagramEntry {
+  percent: Percent
+  path: [Currency, Currency, FeeAmount][]
+  protocol: Protocol
+}
+
+const V2_DEFAULT_FEE_TIER = 3000
+
+export const routerGradientCss = css`
```

Comment:
nit: I think this should be defined in `RoutingDiagram/index.tsx` where it is used. It could also just be inlined there. It's a one-off style, right?

Reply:
yeah its one off here, could def just include in the index file